File: fw-portal-web/src/main/resources/static/resources/js/dto/CidadeDto.js
class CidadeDto{

    _nome;
    _uf;
    _codigo;

    constructor(nome,uf,codigo){
        this._nome = nome;
        this._uf = uf;
        this._codigo = codigo;
    }

    getNome(){
        return this._nome;
    }
    getUf(){
        return this._uf;
    }
    getCodigo(){
        return this._codigo;
    }
}

File: fw-portal-web/src/main/java/com/faraway/fwportal/dto/CidadeDto.java
package com.faraway.fwportal.dto;

import com.faraway.fwportal.model.domain.Cidade;
import com.fasterxml.jackson.annotation.JsonProperty;

public class CidadeDto {

	private String nome;
	private String uf;
	@JsonProperty("codigo")
	private String codigoIbge;

	public CidadeDto(Cidade cidade) {
		super();
		this.nome = cidade.getNome();
		this.uf = cidade.getUf();
		this.codigoIbge = cidade.getCodigoIbge();
	}

	public String getNome() {
		return nome;
	}

	public String getUf() {
		return uf;
	}

	public String getCodigoIbge() {
		return codigoIbge;
	}

}
